iron-menu-behavior.html

```diff
@@ -90,16 +90,25 @@
     },
 
     /**
-     * Resets all tabindex attributes to the appropriate value based on the
+     * Reset attributes of the menu's new/changed collection of items:
+     *
+     * Reset all tabindex attributes to the appropriate value based on the
      * current selection state. The appropriate value is `0` (focusable) for
      * the default selected item, and `-1` (not keyboard focusable) for all
      * other items.
+     *
+     * Reset the aria-selected attribute based on the current selection state.
+     * As _applySelection notes, we need to set aria-selected to false for
+     * unselected items, even in their initial state, not just when they become
+     * deselected.
      */
-    _resetTabindices: function() {
+    _resetItemAttributes: function() {
       var selectedItem = this.multi ? (this.selectedItems && this.selectedItems[0]) : this.selectedItem;
 
       this.items.forEach(function(item) {
-        item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+        var isSelected = item === selectedItem;
+        item.setAttribute('tabindex', isSelected ? '0' : '-1');
+        item.setAttribute('aria-selected', isSelected);
```

Comment:
While this is technically correct, it's worth noting that this will double the number of calls to `setAttribute` when attached. Have we considered the performance implications?